sdk/src/main/scala/io/horizen/AbstractSidechainApp.scala

```diff
@@ -136,8 +135,8 @@ abstract class AbstractSidechainApp
   }
 
   if (!isCSWEnabled) {
-    val sc2scIsActive = sc2scConfigurator.canSendMessages || sc2scConfigurator.canReceiveMessages
-    if (sc2scIsActive) {
+    val epoch = TimeToEpochUtils.timeStampToEpochNumber(params, timeProvider.time())
+    if (Sc2ScUtils.isActive(params, ForkManager.getOptionalSidechainFork[Sc2ScFork](epoch))) {
```

Comment:
At this point the fork manager is not yet initialized, we should move it after the initialization.